src/reconcile/rec_visibility.c

```diff
@@ -9,11 +9,11 @@
 #include "wt_internal.h"
 
 /*
- * __rec_update_durable --
- *     Return whether an update is suitable for writing to a disk image.
+ * __rec_update_stable --
+ *     Return whether an update is stable or not.
  */
 static bool
-__rec_update_durable(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_UPDATE *upd)
+__rec_update_stable(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_UPDATE *upd)
```

Comment:
I'm not sure if this rename is valid, we're only checking whether a given update is durable from the currently running transaction's perspective. This isn't necessarily checking whether an update is stable. That would imply that we were comparing it's durable / commit timestamp with the stable timestamp, no?

Reply:
I think what it does originally is to choose a version to write to disk and it is comparing to the upd's durable timestamp so it's named durable. Now we are not using it to select the version but to track some statistics. Within the check, we do:

if (F_ISSET(r, WT_REC_EVICT))
                ++r->updates_unstable;

So I think may be rename it to update_stable as the code is tracking the number of unstable updates by the look of it.